.github/workflows/dependabot-auto.yml

```
@@ -0,0 +1,90 @@
+        # Run additional security checks
+        uv add --dev safety
+        uv run safety check
+    
+    - name: Auto-approve security updates
+      if: success()
+      run: |
+        echo "🔒 Security update - auto-approving after tests pass"
+        gh pr review --approve "${{ github.event.pull_request.number }}"
+        gh pr merge --auto --merge "${{ github.event.pull_request.number }}"
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)